.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     project: ['./tsconfig.eslint.json'],
     sourceType: 'module',
   },
-  plugins: ['ban', 'filenames', 'local', '@typescript-eslint'],
+  plugins: ['ban', 'filenames', '@kr-modules/coda-rules', '@typescript-eslint'],
```

Comment:
pnpm makes it harder than yarn to use undeclared dependencies, so I moved the local eslint rules into a subpackage